core/src/toga/widgets/webview.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         url: str | None = None,
         content: str | None = None,
         user_agent: str | None = None,
+        on_navigation_starting=None,
```

Comment:
This needs a type declaration (if only for documentation purposes)

Reply:
OK, added type declaration